public/robots.txt

```diff
@@ -2,6 +2,8 @@
 #
 # To ban all spiders from the entire site uncomment the next two lines:
  User-agent: *
- Disallow: /flags
+ Disallow: /admins
+ Disallow: /blockeds
+ Disallow: /activities
 
  Sitemap: http://www.catforum.mntc.in/sitemap.xml
```

Comment:
oh there isn't. We need to fix it. Only admin user should be able to see blocked users and only proper user should be able to see his activity, not everyone.